sciencefixion/glitch_craft
# Google OAuth

## All Submissions:

### New Feature Submissions:
* [X] Have you added an explanation of what your changes do?

### Testing:

**Test coverage** = 100%
--------------------------------------------------------------------------------
### Change Log:

- Add the User model & controller, with User Show view and associated tests including
  * Validations
  * Class Methods
  * Feature test to check if a user can log in with Google OAuth
- Create the Sessions Controller and create action
- Add the helper method current_user in the Application Controller
- Add the mock_user method in the Rails Helper file to mock creating a user with data pulled from Google
